app/views/two_factor_authentication/otp_verification/show.html.erb

```diff
@@ -8,10 +8,19 @@
 
 <% if @presenter.otp_expiration.present? %>
   <%= render CountdownAlertComponent.new(
-        show_at_remaining: IdentityConfig.store.otp_expiration_warning_seconds.seconds,
-        alert_options: { class: 'margin-bottom-4' },
-        countdown_options: { expiration: @presenter.otp_expiration },
+      show_at_remaining: IdentityConfig.store.otp_expiration_warning_seconds.seconds,
+      alert_options: { class: 'margin-bottom-4' },
+      countdown_options: { expiration: @presenter.otp_expiration },
+    ) %>
+
+
+    <p class="usa-sr-only" role="timer" aria-live="polite" aria-atomic="true">
+      <%= render CountdownComponent.new(
+        expiration: Time.zone.now,
```

Comment:
Not sure if this is temporary, but the expiration will need to be in the future matching `@presenter.otp_expiration` above.